src/plugins/nest.js

```diff
@@ -0,0 +1,294 @@
+'use strict'
+
+function createWrapCreate (tracer, config) {
+  return function wrapCreate (create) {
+    return function createWithTrace () {
+      const instance = arguments[0]
+      const methodName = arguments[2]
+
+      const handler = create.apply(this, arguments)
+      return function (req, res, next) {
+        let opName = 'nest.request'
+        const scope = tracer.scope()
+        const childOf = scope.active()
+        const span = tracer.startSpan(opName, {
+          childOf,
+          tags: {
+            'component': 'nest',
+            'nest.request.method': req.method,
+            'nest.request.path': req.route.path
+          }
+        })
+
+        if (instance.constructor && instance.constructor.name) {
+          opName = instance.constructor.name
+        }
+        span.setTag('resource.name', `${opName}.${methodName}`)
```

Comment:
Resource name isn't something we should adopt in new instrumentations.  Please set the operation name directly.